src/Compilers/CSharp/Portable/Symbols/PropertySymbolExtensions.cs

```diff
@@ -14,12 +12,12 @@ internal static class PropertySymbolExtensions
         /// If the property has a GetMethod, return that.  Otherwise check the overridden
         /// property, if any.  Repeat for each overridden property.
         /// </summary>
-        public static MethodSymbol GetOwnOrInheritedGetMethod(this PropertySymbol property)
+        public static MethodSymbol? GetOwnOrInheritedGetMethod(this PropertySymbol? property)
         {
-            while ((object)property != null)
+            while (property is not null)
```

Comment:
>is not [](start = 28, length = 6)

It looks like this PR doesn't follow the general guideline to keep unrelated refactorings out of PRs that change product behavior. Also, in general we prefer avoiding making stylistic changes to existing code. Everybody has personal preferences and if analyzer doesn't flag the code, it is in a good state style/formatting wise.

Reply:
Since this is an ask mode fix, I would prefer changes like that reverted and keeping this PR to a necessary minimum to address the issue.

---
In reply to: [497930116](https://github.com/dotnet/roslyn/pull/48168#discussion_r497930116) [](ancestors = 497930116)

Reply:
These changes are what allowed me to figure out the bug here. My strategy was to nullable enable the callstack, which allowed me to figure out where the bug was coming from. I'd prefer to keep them.

Reply:
> These changes are what allowed me to figure out the bug here. My strategy was to nullable enable the callstack, which allowed me to figure out where the bug was coming from. I'd prefer to keep them.

Changes to comparisons to null and the like are unnecessary. I think they should be reverted and any other changes that are not meant to change product behavior. Nullable annotations and asserts are fine, of course.

---
In reply to: [497940563](https://github.com/dotnet/roslyn/pull/48168#discussion_r497940563) [](ancestors = 497940563)